src/components/MarkSelector.jsx

```diff
@@ -29,7 +30,9 @@ const MarkSelector = (props) => {
               classes="m-auto "
               width={32}
               height={32}
-              color={mark === "x" ? CssVariables.dark : CssVariables.silver}
+              color={
+                mark === "x" ? getMarkColor("x", true) : getMarkColor("o", true)

```

Comment:
Instead:
         `mark === "x" ? getMarkColor("x", true) : getMarkColor("o", true)`
Could be:
            `getMarkColor(mark)`